app/components/affiliate-unit.js

```diff
@@ -29,6 +29,12 @@ export default Component.extend(
       } else if (!this.isInContent && this.unit.links && this.unit.links.search) {
         this.unit.link = this.unit.links.search;
       }
+
+      if (this.unit.campaign === 'ddb') {
```

Comment:
Looks like a hack - is there any way that we can move those params to slots config? (it doesn't have to be right now)

Reply:
How would you see that working?

Reply:
Do you mean continue what chris was doing by expanding the number of links?